src/Public/Send-OSNotification.ps1

```diff
@@ -1,24 +1,33 @@
 <#
 .SYNOPSIS
-Short description
+Displays toast notifications on Linux, macOS, and Windows.
 
 .DESCRIPTION
-Long description
+The Send-OSNotification function displays notification on Linux, macOS, and Windows.
+
+Each operating system uses different methods for generating and displaying notifications, but this function abstracts 
+each of them for a consistent experience regardless of the environment in which it is used.
 
 .PARAMETER Body
-Parameter description
+The main text of your notification. You must specify a body string.
 
 .PARAMETER Title
-Parameter description
+The title of your notification. Generally this is rendered bold at the top of the notification.
+
+Defaults to "PowerShell Notification".
 
 .PARAMETER Icon
-Parameter description
+If specified, the specified picture is included on the notification.
+
+There is a default Icon included on Windows only.
 
 .EXAMPLE
-An example
+Send-OSNotification -Title 'Script Completed' -Body "The script you were running finished at $(Get-Date -Format t)"
+
+This command generates a notification that includes the time at which it was generated in the body.
 
 .NOTES
-General notes
+This function relies on the MacNotify module on macOS, and the PSNotifySend module on Linux.
```

Comment:
Might be worth calling out that 
Notifications are "best effort". Certain features may not be supported on all platforms but PoshNotify will still attempt to send a notification with as much of the information provided as it can.

Reply:
Absolutely! I've "borrowed" your blurb